lib/fastlane/plugin/ddg_apple_automation/helper/asana_helper.rb

```diff
@@ -298,14 +298,17 @@ def self.update_asana_tasks_for_public_release(params)
 
       def self.fetch_tasks_for_tag(tag_id, asana_access_token)
         asana_client = make_asana_client(asana_access_token)
-
         task_ids = []
         begin
           response = asana_client.tasks.get_tasks_for_tag(tag_gid: tag_id, options: { fields: ["gid"] })
           loop do
-            task_ids += response.map(&:gid)
-            response = response.next_page
-            break if response.nil?
+            task_ids += response.data.map(&:gid)
```

Comment:
This returns a wrapped `data` value rather than just an object with gid at the top level